AU2/plugins/CorePlugin.py

```diff
@@ -523,10 +523,16 @@ def gather_config_names(self):
         Gathers the name of all ConfigExports from all plugins
         """
         names = []
-        for p in PLUGINS:
-            for c in p.config_exports:
-                names.append(c.display_name)
-        names.sort()
+        configs = (c for p in PLUGINS for c in p.config_exports)
+        for c in sorted(configs, key=lambda e: e.display_name):
+            disp = c.display_name
+            # display dangerous config options in red if the game has already started
+            if isinstance(c, DangerousConfigExport) and get_game_start() <= get_now_dt():

```

Comment:
`DangerousConfigExport`s should probably make themselves a little annoying to modify after game start. Could we implement a text challenge for all `DangerousConfigExport`s, similar to what is done for `Reset Database` in (iirc) `LocalBackupPlugin`?  You might want to move the challenge into a library location to reuse the functionality across both.

Reply:
I think for this I would want to combine what I wrote above with implementing what you suggested for #57 (letting each options function take the results of all the previous options functions) so that the second options function can do the challenge before `config_ask` is called, in the case that a `DangerousConfigExport` is selected.